src/app/core/services/advance-request.service.ts

```
@@ -142,42 +145,59 @@ export class AdvanceRequestService {
-    return this.apiService.post('/advance_requests/add_approver', data);
+    return this.approverService
+      .post<PlatformApiResponse<AdvanceRequests>>('/advance_requests/add_approver', payload)
+      .pipe(map((response) => response.data));
   }
 
   @CacheBuster({
     cacheBusterNotifier: advanceRequestsCacheBuster$,
   })
-  submit(advanceRequest: Partial<AdvanceRequests>): Observable<AdvanceRequests> {
-    return this.apiService.post('/advance_requests/submit', advanceRequest);
+  submit(advanceRequest: Partial<AdvanceRequests>, isApprover: Boolean): Observable<AdvanceRequestPlatform> {
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)